### DISCUSSION TITLE
lost data every time i update chatwoot

### DISCUSSION THREAD

**Discussion by mredodos** [chatwoot/chatwoot#9315] [Self hosted]
hi, how is possible i lost data every time i upgrade chatwoot? wha i miss? 
how can upgrade chatwoot without lost data ?

**Comment by pranavrajs** [comment]
@mredodos Can you provide more information here? A lot of context is missing. Did you lose the DB or the uploads? How have deployed this?

**Reply by mredodos** [reply]
all the channel not work anymore, some configuration missing

**Reply by KunangKunnang** [reply]
I also experience the same thing every time I update it since the version 3.2 update. I update it manually, by reinstalling the app and updating with the old DB, it works normally for me.

**Reply by mredodos** [reply]
the poroblem is i use always the same DB

it seems the users too lost id or company profile change.

in short I have to reconfigure everything all over again
